src/sentry/models/integration.py

```diff
@@ -52,7 +52,9 @@ class RepositoryProjectPathConfig(DefaultFieldsModel):
 
     repository = FlexibleForeignKey("sentry.Repository")
     project = FlexibleForeignKey("sentry.Project", db_constraint=False)
-    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration", null=True)
+    organization_integration = FlexibleForeignKey(
+        "sentry.OrganizationIntegration", on_delete=models.CASCADE
+    )
```

Comment:
You'll need to generate a migration for this. 

In general, I'd recommend against changing a column to be not null, since it will lock the table while postgres validates. Since this table is small it might not be a big deal, but we'll need to make sure there are no null values in the database before running the query.

Reply:
@wedamija yea the table is pretty small at ~12000 rows and ~43 orphaned rows. So I think we can just run the migrations.RunPython to delete the orphaned rows and then alter the null constraints on the column.